lti/migrations/env.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import with_statement
+import os
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = None
+
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+
+def get_database_url():
+    if 'DATABASE_URL' in os.environ:
+        return os.environ['DATABASE_URL']
+    return config.get_main_option("sqlalchemy.url")
```

Comment:
This is copied from h except that it just returns `os.environ['DATABASE_URL']` directly, it doesn't have a `database_url()` function to pass it through like h does. Which is maybe not ideal but would have to tweak how the config functions work. This should be fine